tests/validation/tests/v3_api/test_import_cluster.py

```diff
@@ -65,15 +78,21 @@ def create_nodes():
 
 def create_rke_cluster_config(aws_nodes):
 
-    configfile = "cluster.yml"
+    # Create RKE Config file
+    print("Number of nodes ")
+    print(AWS_NODE_COUNT)
+    if (AWS_NODE_COUNT == 3):
+        configfile = "cluster.yml"
+    if (AWS_NODE_COUNT == 9):
+        configfile = "cluster_config_1.yml"
```

Comment:
We should be able to generate the config file for any number of worker nodes without having to have any additional config yml. 
Existing config yml can be used for first 3 nodes and for the rest of the nodes build the configuration dynamically without relying on config yml.

Reply:
Done